### PR TITLE
Faq fast follow work

### PR DESCRIPTION
**Pull request by Samcbehrens**
## Work done 

* Animation  on toggle

* allow only one section to be open at a time 
* Reinstitute spanish page to look like old page until we have new translations
Screen Shot 2020-10-04 at 2 53 34 PM

### CODE REVIEW

File: site/src/components/page-faq/question-section.scss [hunk 1]
```diff
@@ -13,13 +13,32 @@
     align-self: center;
 }
 
+section { 
+    max-height: 0; 
+    overflow: hidden; 
+    -webkit-transition: max-height 0.25s ease-in-out;
+    -moz-transition: max-height 0.25s ease-in-out;
+    -o-transition: max-height 0.25s ease-in-out;
+    transition: max-height 0.25s ease-in-out;
```
Comment (nexussays) [inline site/src/components/page-faq/question-section.scss:20]: The SASS parser will take care of adding platform specifics if needed

**Review by nexussays — DISMISSED** [review]
Would recommend changing the url back to `/faq` just for simplicity sake

**Comment by Samcbehrens** [comment]
There is some wonkiness around the animation and autoclosing sections as pictured below. I will remove the logic to autoclose sections logic so we can keep the animation looking smooth.

File: site/src/components/page-faq/question-section.scss (hunk 1 above)
Comment (Samcbehrens) [reply site/src/components/page-faq/question-section.scss:20]: nice thanks!

**Review by nexussays — APPROVED** [review]

**Pull request merged**